draft-ietf-rtcweb-jsep.xml

```diff
@@ -137,7 +139,9 @@ ipr="trust200902">
         only the implementation has the necessary knowledge of
         candidates and other transport info. Performing this separation
         also provides additional flexibility; in protocols that
-        decouple session descriptions from transport, such as Jingle,
+        decouple session descriptions from transport, such
```

Comment:
This still feels clunky because you talk about "SIP with trickle ICE" as a case where transports are decoupled, but then mention SIP in the sentence below.

I think you probably want to reword this - will try to make a suggestion once I finish reviewing the other PRs.